pkg/builder/builder.go

```diff
@@ -368,6 +370,65 @@ func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
 	return true
 }
 
+func getDepsFiles(deps []string) []string {
```

Comment:
Is it possible to add the unit test cases for this func?

Reply:
I am working on it.